ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -956,15 +956,15 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "ID": "addOrRemoveLegalCounsel",
-    "Name": "Add or remove legal counsel",
-    "Description": "Add or remove legal counsel",
+    "Name": "Add or remove counsel",
+    "Description": "Add or remove counsel",
     "DisplayOrder": 11,
     "PreConditionState(s)": "Submitted;GATEKEEPING_LISTING;Gatekeeping;PREPARE_FOR_HEARING;FINAL_HEARING",
     "PostConditionState": "*",
     "SecurityClassification": "Public",
-    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-legal-counsel/about-to-start",
-    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-legal-counsel/about-to-submit",
-    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-legal-counsel/submitted",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-counsel/about-to-start",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-counsel/about-to-submit",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/manage-counsel/submitted",
```

Comment:
I think technically on the pipeline the CCD definition is uploaded a little before the pods refresh with the new code, so in theory there could be a small gap where these links fail to follow through (not sure how much use this event gets, but might be worth just mentioning for when we deploy (or we could leave the old urls potentially), if anyone reports an error!)